src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -79,6 +80,8 @@ class DraftJSEditor extends React.Component<Props> {
 
     labelID = uniqueId('label');
 
+    commentFormTip = uniqueId('comment-form-tip');
```

Comment:
Can we genericize this variable name and unique id prefix, since this component can be used in many places? Can we end it in `ID` to stay consistent with `labelID` above?